config/istio/istio-generated/xxx-generated-istio.yaml

```diff
@@ -5308,7 +5309,7 @@ data:
           "excludeIPRanges": "",
           "excludeInboundPorts": "",
           "excludeOutboundPorts": "",
-          "holdApplicationUntilProxyStarts": true,
+          "holdApplicationUntilProxyStarts": false,
```

Comment:
It's weird that the generated config becomes false here. What you configured looks correct, though, so I assume this still works. 👍